Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -866,6 +867,16 @@ ErrorOr<NonnullRefPtr<GUI::Action>> HackStudioWidget::create_remove_current_edit
     });
 }
 
+ErrorOr<NonnullRefPtr<GUI::Action>> HackStudioWidget::open_file_in_single_click()
+{
+    return GUI::Action::create_checkable("&Toggle Open File In Single Click", [this](auto&) {
+        if (!m_project_tree_view->activates_on_selection())
+            m_project_tree_view->set_activates_on_selection(true);
+        else
+            m_project_tree_view->set_activates_on_selection(false);
```

Comment:
```cpp
m_project_tree_view->set_activates_on_selection(!m_project_tree_view->activates_on_selection());
```

Reply:
Updated function/variables name to match with Serenity OS style convention. :)